src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -363,4 +363,7 @@
   <data name="PX1037" xml:space="preserve">
     <value>UnboundDacFieldInKeyDeclaration</value>
   </data>
+  <data name="PX1076" xml:space="preserve">
+    <value>CallToApiMarkedWithPXInternalUseOnly</value>
```

Comment:
@EAndrosova please review diagnostic short name

Reply:
`PXInternalUseOnly` is, basically, a matter of implementation that might be changed later. I suggest "CallToInternalApi" or something like that

Reply:
fixed